app/Http/Controllers/License/LicenseController.php

```diff
@@ -205,17 +204,23 @@ public function updateLicensedDomain($licenseCode, $domain, $productId, $license
      */
     protected function getIpAndDomain($domain)
```

Comment:
This method can be made more readable. My suggestion:
```
        if($domain != ""){
            if(ip2long($domain)){
                return ["ip"=> $domain, 'domain'=> "", 'requiredomain'=>0]
            } else {
                return ["ip"=> "", 'domain'=> $domain, 'requiredomain'=>1]
            }
        }

        return ['ip'=> "", 'domain'=> $domain, 'requiredomain'=> 0];
```
Add necessary comments to it